mitmproxy/contentviews/image/view.py

```diff
@@ -19,6 +22,11 @@ class ViewImage(base.View):
     ]
 
     def __call__(self, data, **metadata):
+        if imghdr.what('', h=data) == 'png':
+            f = "PNG"
+            parts = image_parser.parse_png(io.BytesIO(data))
+            fmt = base.format_dict(multidict.MultiDict(parts))
+            return "%s image" % f, fmt
```

Comment:
I was thinking it'll look like this -
```
if 'png':
    parse_png()
elif 'jpeg':
    parse_jpeg()
fmt = format_dict()
return "%s image" % f, fmt
```
Otherwise I'll have to add a `return` inside every `if`.